OpenKeychain/src/main/java/org/sufficientlysecure/keychain/util/Preferences.java

```diff
@@ -362,62 +361,32 @@ public void setKeySignaturesTableInitialized() {
         editor.commit();
     }
 
-    public static class CloudSearchPrefs implements Parcelable {
-        public final boolean searchKeyserver;
-        public final boolean searchKeybase;
-        public final boolean searchFacebook;
-        public final boolean searchWebKeyDirectory;
-        public final HkpKeyserverAddress keyserver;
+    @AutoValue
+    public static abstract class CloudSearchPrefs implements Parcelable {
+        public abstract boolean searchKeyserver();
```

Comment:
I didn't have better name for these accessor methods... `isSearchKeyserver`? `isSearchingKeyserver`? `shouldSearchKeyserver`? If you've got better idea tell me and I'll adjust the code.

Reply:
Perhaps isKeyserverEnabled?